src/Missions.java

```diff
@@ -0,0 +1,125 @@
+import java.util.*;
+
+public class Missions {
+    public static Scanner input = new Scanner(System.in);
+
+    public static double average(double... numbers) {
+        double sum = 0;
+        for (double number : numbers) {
+            sum += number;
+        }
+        return sum / numbers.length;
+    }
+
+    public static void printAverage(double number1, double number2, double number3) {
+        System.out.println(average(number1, number2, number3));
+    }
+
+    public static boolean isEven(double number) {
+        return number % 2 == 0;
+    }
+
+    public static void isBothEven(double number1, double number2) {
+        if (isEven(number1) && isEven(number2)) {
+            System.out.println("Both numbers are even");
+        } else {
+            System.out.println("Womp Womp not both of the numbers are even");
+        }
+    }
+
+    public static void isBiggerThenAverage(double number1, double number2) {
```

Comment:
Is should return Boolean (you can call it ex) but if you want to go with informative name u should return and put print in other function